app/assets/stylesheets/application.scss

```diff
@@ -12,4 +12,32 @@
  *
  *= require_tree .
  *= require_self
+ *= require foundation_and_overrides
  */
+
+.top-bar {
+   margin-bottom: 30px;
+}
+
+.name {
+   display: inline;
+}
+
+.book {
+  padding: 0;
+  height: 240px;
+  text-align: center;
+  h4 {
+    margin-top: 2em;
+  }
+}
+
+.dashed {
+   border: 2px dashed gray;
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 3 spaces